app/assets/stylesheets/application.css.scss

```diff
@@ -66,6 +66,7 @@ main {
   }
 }
 
+#contact_form,
```

Comment:
Avoid using id selectors<br>Selector `contact_form` should be written in lowercase with hyphens